components/VC/Views/VCCardView.tsx

```diff
@@ -68,27 +72,60 @@ export const VCCardView: React.FC<VCItemProps> = props => {
             ? Theme.Styles.selectedBindedVc
             : Theme.Styles.closeCardBgContainer
         }>
-        <VCCardViewContent
-          vcMetadata={props.vcMetadata}
-          context={context}
-          walletBindingResponse={walletBindingResponse}
-          credential={vc}
-          verifiableCredentialData={verifiableCredentialData}
-          fields={fields}
-          wellknown={wellknown}
-          generatedOn={formattedDate}
-          selectable={props.selectable}
-          selected={props.selected}
-          service={service}
-          isPinned={props.isPinned}
-          onPress={() => props.onPress(service)}
-          isDownloading={props.isDownloading}
-          flow={props.flow}
-          isKebabPopUp={isKebabPopUp}
-          DISMISS={DISMISS}
-          KEBAB_POPUP={KEBAB_POPUP}
-          isVerified={props.vcMetadata.isVerified}
-        />
+        {props.isTopCard ? (
+          <CopilotStep
```

Comment:
We are calling CopilotStep component of react-native-copilot in multiple places and everywhere we are passing key, text, name, order and child component to be rendered inside copilotView. Shall we check if it is possible to create a separate component and pass these data as props to this new component and the new component will render the data after applying the styles. **(similar to Text component)**

Don't pass styles as props as we are applying different styles in different places and if we pass the styles as props it will make the customization difficult may be pass only other props and pass the children component that needs to be rendered.

Reply:
create a copilot component